api/v1/db/oracledb/member-dao.js

```diff
@@ -168,6 +169,33 @@ const deleteMember = async (memberId) => {
   }
 };
 
+const databaseName = string => (decamelize(string).toUpperCase());
+
+const isTruthyOrZero = val => (val || val === 0);
+
+const patchMember = async (memberId, attributes) => {
+  const connection = await conn.getConnection();
+  try {
+    const filteredAttributes = _.pickBy(attributes, isTruthyOrZero);
+    const joinedStringArray = _.map(filteredAttributes,
+      (value, key) => (`${isTruthyOrZero(value) ? `${databaseName(key)} = :${key}` : ''}`));
+    const joinedString = _(joinedStringArray).compact().join(', ');
+    const sqlQuery = `
+    UPDATE MEMBERS
+    SET ${joinedString}
+    WHERE MEMBER_ID = :id
+    `;
+    if (_.isEmpty(filteredAttributes)) {
+      return true;
+    }
```

Comment:
Similar comment in another PR, could you remove this check directly after `filteredAttributes` is defined?